cmd/influxd/server_integration_test.go

```diff
@@ -1458,9 +1458,10 @@ func Test3NodeServerFailover(t *testing.T) {
 }
 
 // ensure that all queries work if there are more nodes in a cluster than the replication factor
-func Test3NodeClusterPartiallyReplicated(t *testing.T) {
+// and there is more than 1 shards
+func Test6NodeClusterPartiallyReplicated(t *testing.T) {
 	t.Parallel()
-	testName := "3-node server integration partial replication"
+	testName := "6-node server integration partial replication"
```

Comment:
What we are trying to do is get the server to create more than one shard per shard group, to test the full range of possibilities across writing/reading series data.

From what we could tell, the number of shards that a shard group creates is calculated on the # of data nodes / replication factor.  So, we needed something that would do that.  We probably should make this a 5 node cluster with a replication factor of 2.